app/helpers/website_schedule_helper.rb

```diff
@@ -0,0 +1,9 @@
+module WebsiteScheduleHelper
+
+  def schedule_for_day(schedule, day_number)
+    schedule.select { |program_session| program_session[:conference_day] == day_number }
```

Comment:
@codyjamesbrooks given that we now have the advantage of having direct access to the data I would lean towards trying to do more of the querying using ActiveRecord. We can look at this together if you like.

Reply:
As an FYI for anyone that is following along. I will work on moving this down to the database layer. 
For the time being this method is still required, BUT with the help of @jonsgreen I removed the TimeSlotSerlizer from the controller.

For the time being this method is working just fine, but it is performing work that is best left to a db